.github/workflows/build-and-upload.yml

```diff
@@ -139,7 +139,8 @@ jobs:
       - name: Build Binaries
         run: |
           go mod download
-          export GOARCH=arm64 && make targetallocator 
+          go mod tidy
```

Comment:
Should we do `go mod tidy` before `go mod download` ?

Reply:
I think `go mod tidy` should be second, since it is mainly cleanup